tools/online_editor/src/index.ts

```diff
@@ -28,6 +44,29 @@ import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
 
 import slint_init, * as slint from "@preview/slint_wasm_interpreter.js";
 
+class LogOutputChannel implements OutputChannel {
+  constructor(
+    private readonly logger: (value: string) => void,
+    readonly name: string
+  ) {}
+
+  append(value: string) {
+    this.logger(this.name + "(append):" + value);
```

Comment:
I don't understand this...   Where is this shown?  Is that meant to be in production code?

Reply:
It can use that to plug it in as "output channel" into places and get those to log to the console.

No, this should not be in production code, but I will need it for a bit longer.

Reply:
Removed